docs/contributing/submit-to-starter-library.md

```diff
@@ -1,55 +1,59 @@
 ---
-title: Submit to Starter Library
+title: Enviar para Biblioteca Starter
 ---
 
-Have you created a Gatsby starter you'd like to add to the [Starter Library](/starters/)? Follow these instructions.
+Você criou um Gatsby _starter_ que gostaria de adicionar na [Biblioteca _Starter_](/starters/)? Siga estas instruções.
 
-## Steps
+## Etapas
 
-To get your site added to the starter library, follow the two steps below.
+Para adicionar seu site à biblioteca _starter_, siga as duas etapas abaixo.
 
-1.  If this is your first contribution to the Gatsby open source repo, follow the [Contribution guidelines](/contributing/code-contributions/).
+1. Se esta é sua primeira contribuição ao repositório _open source_ do Gatsby, siga as [Orientações de contribuição](/contributing/code-contributions/).
```

Comment:
Prefiro "essa é a sua" do que "esta", independente da normal culta. Fica mais fluida a leitura.

Reply:
é padrao comecar  o link com  letra maiuscula? Se nao for , minuscula melhora a leitura.